README.md

```diff
@@ -37,6 +37,8 @@ version = 1.0.0-Beta-01
 ```
 
 **In Maven**:
+
+Note that the jar is in beta and not available yet in public maven. clone this repository and build it localy to add it to your local maven repository, or use the [pre-built jars](lib/)
```

Comment:
-> Note: (add the '->' is for .md formatting)

. clone - . Clone (capitalize)

Reply:
done.